src/main/java/com/github/dockerjava/api/model/BindOptions.java

```diff
@@ -0,0 +1,44 @@
+package com.github.dockerjava.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.dockerjava.core.RemoteApiVersion;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import java.io.Serializable;
+
+/**
+ * @since {@link RemoteApiVersion#VERSION_1_24}
```

Comment:
Now I've used the link everywhere already, and it's not slower because I copy-paste anyway.